Added a new option to toggle Always on Top (floating window).

ljunkie opened the pull request:
First off, I just wanted to say that this tool is fantastic, and I’m excited to see where it goes!

This PR adds an option to toggle the 'Always on Top' feature. This feature gives users more control over their experience by allowing them to choose whether they want the window to stay on top, while maintaining the default behavior. This also makes the capture window shareable during meetings, as platforms like Google Meet do not include windows that are always on top.

I hope this PR comes across in the right spirit, just trying to contribute and not step on any toes. Let me know what you think!

lvcabral commented on public/main.js:
```diff
@@ -140,7 +148,7 @@ app.whenReady().then(async () => {
   const mainWindow = createMainWindow();
   const displayWindow = createDisplayWindow();
-  displayWindow.setAlwaysOnTop(true, "floating", 1);
+  displayWindow.onAlwaysOnTop(settings.display.alwaysOnTop);
```
Please keep the parameters `floating` (status) and `1` (relativeLevel) when the `alwaysOnTop`) is ``true`.
When is `false` just pass the flag not other parameters needed.
https://www.electronjs.org/docs/latest/api/browser-window#winsetalwaysontopflag-level-relativelevel

ljunkie replied:
👋 this was a new method ([onAlwaysOnTop](https://github.com/lvcabral/carabiner/pull/1/commits/bcf82eea7742f35e29c81150647984a9a49fe5b8#diff-bea8b3046e769bb45b7b3df65d7b80dde75f65af36516f2a894ede0c97c2b978R55-R62)) that continue to pass the params. 

I see that isn't best practice/maintainable, so I've opted to move it into a helper in 140fc260f06607ac32545623f5640bdc63f378d4

lvcabral commented on public/main.js:
```diff
@@ -260,6 +268,13 @@ app.whenReady().then(async () => {
+    settings.display.alwaysOnTop = alwaysOnTop;
+    saveSettings(settings);
+
+    displayWindow.onAlwaysOnTop(alwaysOnTop);
```
Same here, pass the parameters, maybe create a function and use in both places.

lvcabral reviewed (changes requested):
Thanks very much for the contribution, I was thinking about adding this flag.
Just have a change request, take a look.

ljunkie commented:
> Thanks very much for the contribution, I was thinking about adding this flag. Just have a change request, take a look.

You're welcome! I'm glad to hear you were already considering adding this flag. I've addressed the changes, moved the extended functionality into a helper, and updated the PR. Let me know if there's anything else you'd like me to tweak

lvcabral reviewed (approved):
Looks good now!
